repos.bzl

```diff
@@ -2187,3 +2179,11 @@ def go_repositories():
         sum = "h1:u2CU3YKy9I2pmu9pX0eq50wCgjfGIt539SqR7FbHiho=",
         version = "v1.0.4",
     )
+    go_repository(
+        name = "com_github_knative_build",
```

Comment:
Idelaly this will stay above the `com_github_knative_pkg repo` where it has been.

Reply:
done